### DISCUSSION TITLE
编译警告求助

### DISCUSSION THREAD

**Discussion by hellokai55** [didi/booster#443] [Q&A] [answered]
AGPBI: {"kind":"warning","text":"Expected stack map table for method with non-linear control flow.","sources":[{"file":"xxxx/app/build/intermediates/transforms/booster/debug/189.jar"}],"tool":"D8"}

执行:app:dexBuilderDebug时会报这个警告很多条，想问下这个是为啥啊，能解决吗？

**Comment by johnsonlee** [comment]
估计是这个 jar 被 proguard 优化过

**Reply by hellokai55** [reply]
大佬这个需要处理吗，还是不用关心呢？还有一个问题https://github.com/didi/booster/issues/465 不会处理...这个怎么排查呢，想请大佬指教下。

**Reply by johnsonlee** [reply, marked as answer]
可忽略掉
